packages/docs/docs/getting-started/faqs.md

```diff
@@ -0,0 +1,28 @@
+---
+
+title: FAQs
+---
+
+# ❓FAQs
+
+What is Unirep?
+
+* UniRep is an anonymous reputation protocol. The protocol allows developers to build applications that associate data with anonymous users. 
+
+What is an epoch key?
+
+* Epoch Keys are how a user stays anonymous while receiving data. An epoch key is a temporary anonymous identity or persona that a user uses to interact with a Unirep application across epochs that are renewed each epoch.
+
+What is the architecture of a Unirep application?
+
+* A Unirep application will have the architecture of web app <-> relay <-> blockchain. 
+
+How can I get started building on Unirep?
+* Refer to our 'Getting Started' section on how to bootstrap a UniRep application.
```

Comment:
you can link to the Getting Started section by
```
[Getting started with create-unirep-app](https://developer.unirep.io/docs/getting-started/create-unirep-app)
[Getting tarted with typescript/javascript](https://developer.unirep.io/docs/getting-started/ts-js)
```
preview
[Getting started with create-unirep-app](https://developer.unirep.io/docs/getting-started/create-unirep-app)
[Getting tarted with typescript/javascript](https://developer.unirep.io/docs/getting-started/ts-js)

Reply:
also if you mention about epoch key or attester you can also use the link like this